### PR TITLE
Add ajax-powered select elements for numismatic issues and coins

### LINKED ISSUE #2715: Server-powered typeahead for relations on Numismatic resource forms
Lookup values instead of pre-loading all of them, because that's really slow.

### PR DESCRIPTION
**Pull request by eliotjordan**
Co-authored-by: James Griffin <jrgriffiniii@gmail.com>

Closes #2715

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -61,3 +62,39 @@
...
+.v-select .dropdown li a {
+  padding: 10px 20px;
+  width: 100%;
+  font-size: 1.25em;
+  color: #3c3c3c;
+}
+
+.v-select .dropdown-menu .active > a {
+  color: #fff;
+}
```
Comment (eliotjordan) [inline app/assets/stylesheets/application.scss]: I think the css should probably go into the ajax-select component.

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -17,15 +19,23 @@ document.addEventListener('DOMContentLoaded', () => {
   // Set CSRF token for axios requests.
   axios.defaults.headers.common['X-CSRF-Token'] = document.querySelector('meta[name="csrf-token"]').getAttribute('content')
   var elements = document.getElementsByClassName('lux')
-  for(var i = 0; i < elements.length; i++){
-    new Vue({
+  for (var i = 0; i < elements.length; i++) {
+    window['vm' + i] = new Vue({
```
Comment (eliotjordan) [inline app/javascript/packs/application.js]: Probably don't need this. It was an experiment.

File: config/webpacker.yml [hunk 3]
```diff
@@ -36,7 +36,7 @@ development:
   compile: true
 
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  check_yarn_integrity: true
+  check_yarn_integrity: false
```
Comment (eliotjordan) [inline config/webpacker.yml]: Probably don't need this either.

Comment (jrgriffiniii) [reply config/webpacker.yml]: Yes, I suspect that it might be advisable to have this disabled before this PR is ready for review.

**Comment by eliotjordan** [comment]
Tested in staging.

Coin Edit page: 15.5 seconds before, 200ms now.
Issue Edit page: 2.6 seconds before, 700ms now.

**Pull request merged**